<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weather</class>
 <widget class="QMainWindow" name="weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>weather</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="cityLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>城市：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>南京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上海</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北京</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>日期：</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>第一天</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>第二天</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>第三天</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>第四天</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>第五天</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>第六天</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>天气</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>温度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>风速</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="showDate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>249</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>指数</string>
    </property>
    <widget class="QLabel" name="wearLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>穿衣指数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="wearShow">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="carLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>洗车</string>
     </property>
    </widget>
    <widget class="QLabel" name="exLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>晨练</string>
     </property>
    </widget>
    <widget class="QLabel" name="comfortLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>舒适指数</string>
     </property>
    </widget>
    <widget class="QLabel" name="driyingLabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>晾晒</string>
     </property>
    </widget>
    <widget class="QLabel" name="ultraLabel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>紫外线</string>
     </property>
    </widget>
    <widget class="QLabel" name="carShow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="exShow">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="comfortShow">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="dryingShow">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="ultraShow">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
